### PR TITLE
Project Description

### PR DESCRIPTION
**Pull request by Dominik-Maj**
We are adding some `new basic description` of this project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
 # Rock Paper Scissors - Git
 
-Some description.
\ No newline at end of file
+We are learning **git** & __GitHub__.
+
+`hello from vim`
+
+> :wq
```
Comment (Dominik-Maj) [inline README.md:7]: Escaping from Vim

**Review by Dominik-Maj — COMMENTED** [review]
LGTM!

File: README.md (hunk 1 above)
Comment (Eghizio) [reply README.md:7]: Let's fukinnn goooo 🚀

**Review by Eghizio — APPROVED** [review]
Looks good. Let's merge it and deploy it to GitHub Pages! 👌

**Pull request merged**